RecoMuon/TrackingTools/interface/MuonTrackFinder.h

```diff
@@ -61,16 +62,16 @@ class MuonTrackFinder {
   void setEvent(const edm::Event&);
 
   /// convert the trajectories into tracks and load them in to the event
-  edm::OrphanHandle<reco::TrackCollection> load(const TrajectoryContainer&, edm::Event&, const TrackerTopology& ttopo);
+  edm::OrphanHandle<reco::TrackCollection> load(TrajectoryContainer&, edm::Event&, const TrackerTopology& ttopo);
```

Comment:
Passing the container as `const` was not appropriate as the function deletes items held by the container. Although C++ allows that, it is not strictly const correct.